DPG/Tasks/AOTTrack/PID/qaPIDTPC.cxx

```diff
@@ -76,14 +79,17 @@ struct tpcPidQa {
   static constexpr std::string_view hsignal_wTOF[Np] = {"wTOF/signal/El", "wTOF/signal/Mu", "wTOF/signal/Pi",
                                                         "wTOF/signal/Ka", "wTOF/signal/Pr", "wTOF/signal/De",
                                                         "wTOF/signal/Tr", "wTOF/signal/He", "wTOF/signal/Al"};
+  static constexpr std::string_view hnsigma_eta_wTOF[Np] = {"wTOF/nsigma/eta/El", "wTOF/nsigma/eta/Mu", "wTOF/nsigma/eta/Pi",
```

Comment:
hnsigma_eta -> hnsigma_p_eta_Ncl
nsigma/eta (in path) --> nsigma/sparsePinEtaNcl